cpp/include/cudf/types.hpp

```diff
@@ -114,6 +114,14 @@ size_type distance(T f, T l)
  */
 static constexpr size_type UNKNOWN_NULL_COUNT{-1};
 
+/**
+ * @brief Indicates the whether a scalar is valid or not
+ */
+enum class scalar_validity : bool {
+  VALID,   ///< Scalar is valid
+  INVALID  ///< Scalar is invalid
+};
```

Comment:
Is this really needed?

Reply:
My vote is no. A bool is pretty clear in this case.